rdrf/rdrf/views/form_view.py

```diff
@@ -299,7 +299,12 @@ def _enable_context_creation_after_save(
         if not patient_model.in_registry(registry_model.code):
             raise Http404
 
-        if not user.can_view(form_model):
+        form_permission = user.get_form_permission(form_model)
+        if not form_permission.can_view():
+            if form_permission == UserFormPermission.FORM_NOT_TRANSLATED:
+                return render(
+                    request, "rdrf_cdes/form_error_not_translated.html"
```

Comment:
It just occurred to me that the `get` method and here should use the [406 status code ](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/406)

Reply:
I've updated the render response to return a 406. Did you want to have another look, is this what you were expecting?